### ISSUE TITLE
Deployed quartz layout/design?

### ISSUE THREAD

**Issue by DefenderOfBasic** [DefenderOfBasic/in-good-faith-handbook#8]
I think having the homepage be the table of contents is good? (or could have a "cover" page and it can have a "NEXT" be a preface and a table of contents next?)

I think the sidebar should be individual links, NOT folders? (so they're like chapters). And each can be an index page itself?

I want each page to have a NEXT / PREV. So it feels like reading a book. (maybe that's something we can automate in the quartz deploy?)

cc @tonyaajjackson

**Comment by tonyaajjackson** [comment]
There's a question of what trajectories we want people to take through the content. VisakanV had a semi-linear approach in Introspect, where you can read it back to front but it also has Obsidian-style links to other chapters forward and backward throughout the text.

Related thought around rhizomatic idea structures from Jacky Zhao:
https://jzhao.xyz/thoughts/rhizomatic-vs-arborescent

**Comment by DefenderOfBasic** [comment]
> It would be nice to have a little blurb talking about what the project is on Quartz too for first visitors.

yeah I agree! that's what I was thinking the "cover page" could have

> We can hand-roll them relatively quickly though for a small number of pages and chapters

sounds good! can also just do "NEXT" to keep it simple. Can always hit the back button/go back to the chapter index page

> you can read it back to front but it also has Obsidian-style links to other chapters forward and backward throughout the text

oh this is cool!! yeah it reminds me also of this little preface "Is Water H20" gives to the structure of the book. It's a plain old book but he's giving you some tips here on how he wrote it, so that you can choose to skip around or find the parts you need. Like starting with a surface level description of everything, then "zooming in"

<img src="https://github.com/user-attachments/assets/26671151-b9a7-424c-94a4-4f0be5f4acfa" width="450" />

**Comment by DefenderOfBasic** [comment]
it's on my TODO to come up with my own outline/TOC from scratch, and open a PR and compare it with yours/get your feedback. Off the top of my head I'm imagining it as:

- An intro about the book & why we decided to write it etc. 
- A theory section. Each chapter is a specific theory, how to apply it, its failure modes, etc. And maybe one concrete case?
  - I don't have a lot of "background research" but this would be where it goes. Like, I have my own ideas I developed from first principles, would be cool to see if there's real research behind them.  
- A Case Studies section. Where each chapter is a specific real world case study, applying usually many of the theories discussed, or analyzed why it failed
  - we'll put in some OK ones in there and it would be really cool if people contributed, and we selected the best
  - this would be the easiest thing for people to contribute. There's no shortage of people complaining about how social media sucks/is toxic/almost everyone has lots of negative interactions. So we ask people to submit these
  - OR for initial readers who try the draft and try it out in the real world, can submit a writeup and we can collectively analyze it

**Comment by DefenderOfBasic** [comment]
or, at least that's the outline for the physical handbook that I want to publish. The online book can be more nonlinear, and can have a lot more case studies and a lot more theory. Maybe as we write the theory chapters a pattern will emerge? Like these techniques are all about understanding, these techniques are all about diffusing anger, these ones are about changing minds (that's just one possible categorization)

**Comment by tonyaajjackson** [comment]
Sounds great! I wrote the outline as a quick skeleton to hang ideas off of, but fully expect it to reshape and change as we write more pieces. I also love the multi-slice approach to organizing the content - looking forward to playing with those slices and seeing how recontextualizing the chunks of book changes the feel and flow.

**Comment by DefenderOfBasic** [comment]
actually I think I will prototype mdBook. I think quartz is nice as a "notebook" but I really like mdBook's layout & design, it already is 100% what we want out of the box, and has a lot of extensions: https://rust-lang.github.io/mdBook/

**Comment by DefenderOfBasic** [comment]
I think really _this_ is the book design I want: https://thenetworkstate.com/
